__init__.py

```diff
@@ -542,9 +546,19 @@ def get_dashboard_cards(self, message=None):
         cards = self.dashboard_handler.get_collection()
         collection = {"collection": cards}
         return collection
-    
 
+    ######################################################################
+    # Handle Screenshot
+
+    @intent_file_handler("take.screenshot.intent")
+    def take_screenshot(self, message):
+        folder_path = os.path.expanduser('~') + "/Pictures"
```

Comment:
Could this have a config override? On a dev machine, I'd want to put these in a subfloder

Reply:
a config has been added